sites/helpers/requests.py

```diff
@@ -80,3 +91,9 @@ def __init__(self, external_ids: List[str], url: str, msg: str = "") -> None:
         self.external_ids = external_ids
 
     pass
+
+# TODO: Once merging SS, combine this with SS ArticleBatchScrapingError
```

Comment:
📌 Guess it's a good time to tackle this to-do. I'll address this in a separate PR.